### ISSUE TITLE
Since Symfony 3.0.2, the twig->render method doesn't work with absolute path

### ISSUE THREAD

**Issue by Congelli501** [symfony/symfony#17777]
Environment:

- PHP 5.6
- GNU/Linux
- Symfony 3.0.2 (ok with 3.0.1)

Since Symfony 3.0.2, the Symfony\Component\Templating\EngineInterface::render method doesn't work with absolute path to the template file (Unable to find template error).

Step to reproduce (from sratch):

```bash
# Install a base sf project
wget https://symfony.com/installer
chmod +x installer
./installer new testp
cd testp/

# Use an absolute path to a twig template
cat > src/AppBundle/Controller/DefaultController.php << 'EOF'
<?php

namespace AppBundle\Controller;

use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
use Symfony\Bundle\FrameworkBundle\Controller\Controller;
use Symfony\Component\HttpFoundation\Request;

class DefaultController extends Controller
{
    /**
     * @Route("/", name="homepage")
     */
    public function indexAction(Request $request)
    {
        $absPath = $this->get('kernel')->getRootDir() . '/Resources/views/default/index.html.twig';

        // replace this example code with whatever you need
        return $this->render($absPath, [
            'base_dir' => realpath($this->getParameter('kernel.root_dir').'/..'),
        ]);
    }
}
EOF


# Test with symfony 3.0.2
php bin/console server:run

## Go to http://127.0.0.1:8000 --> Unable to find template "/.../testp/app/Resources/views/default/index.html.twig".

# Switch to symfony 3.0.1
patch -p0 --batch --forward composer.json << 'EOF'
--- composer.json.1	2016-02-12 12:57:16.729074815 +0100
+++ composer.json	2016-02-12 12:57:26.717103221 +0100
@@ -18,7 +18,7 @@
     },
     "require": {
         "php": ">=5.5.9",
-        "symfony/symfony": "3.0.*",
+        "symfony/symfony": "3.0.1",
         "doctrine/orm": "^2.5",
         "doctrine/doctrine-bundle": "^1.6",
         "doctrine/doctrine-cache-bundle": "^1.2",
EOF

composer update

# Test with symfony 3.0.1
## Go to http://127.0.0.1:8000 --> Ok
```

**Comment by xabbuh** [comment]
Looks like this is caused by #15272 (see #17683).

Status: Reviewed

ping @jakzal

**Comment by jakzal** [comment]
I started looking into this.

re https://github.com/symfony/symfony/pull/12894#issuecomment-68255459

**Comment by jakzal** [comment]
I won't be able to look into this before the weekend.

**Comment by wesleylancel** [comment]
I'm seeing a regression where the latest 2.8.x-dev is no longer able to parse the following snippet, whereas 2.8.2 is. Possibly the same cause?

`{% extends "@FOSUserBundle/Resources/views/layout.html.twig" %}`

**Comment by xabbuh** [comment]
@wesleylancel What error exactly do you experience?

**Comment by wesleylancel** [comment]
@xabbuh: I'll look up the exact error tomorrow but it says it can't the template in views/@FOSUserBundle or something. Before the PR it simply didn't parse it because the regexp didn't match and it was passed off to Twig, now it does but it can't actually get all the parts correctly so the TemplateReference contains incorrect data.

**Comment by francoispluchino** [comment]
@wesleylancel The list of affected versions of Symfony is listed in #17683.

**Comment by jakzal** [comment]
I'm looking into fixing this now. @wesleylancel I'd appreciate if you shared the stack trace

**Comment by francoispluchino** [comment]
@jakzal, @fabpot You close this issue while the PR #17894 is not merged into the `3.0.x` branch (and `^2.3)`. Is it not better to keep this issue open until all the branches be merged?

**Comment by jakzal** [comment]
@francoispluchino in our workflow issues are closed as soon as a related PR is merged. There's no need to keep them open. Older branches are merged into newer ones on a regular basis.

**Comment by francoispluchino** [comment]
@jakzal Ok, thank you for the precisions!

**Comment by Babacooll** [comment]
@xabbuh  @jakzal I'm sorry to bother you with this, has this been fixed in Symfony 2.8 ? I've seen a release for Symfony 2.3 but not higher. Thanks !

**Comment by jakzal** [comment]
Indeed, only Symfony 2.3 was released recently. Don't worry though, 2.7-3.0 will be released soon and will include this fix.
